rust/kernel/bindings.rs

```diff
@@ -13,7 +13,7 @@
 )]
 mod bindings_raw {
     use crate::c_types;
-    include!(env!("RUST_BINDINGS_FILE"));
+    core::include!(env!("RUST_BINDINGS_FILE"));
```

Comment:
What required this change?

Reply:
Hoped that it would fixe the bindings_raw autocompletion, but sadly does not look like it.

Reply:
removed it again